src/client/components/ChannelInformation/ChannelInnformation.js

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import './ChannelInformation.style.css';
 
 export const RenderChannelInformation = ({
-  headline = 'Here is channel info',
+  channelImg,
```

Comment:
Small nitpick - this component already renders channel information. The naming should be `image` and `title` since it is more natural.